### PR TITLE
Added a how-to-guide for parameter schedulers 

### PR DESCRIPTION
**Pull request by divyanshugit**
This PR is a modified version of [PR#49](https://github.com/pytorch-ignite/examples/pull/49)

@Priyansi @sdesrozis Can you please give a review?

### CODE REVIEW

**Comment by sdesrozis** [comment]
Why not improve #49 instead of creating a new PR ?

**Comment by sdesrozis** [comment]
I feel doubtful about the added description. It’s copy / paste from official PyTorch doc and I’m not sure that some details really matter. I would have preferred something more personal for the project.

File: how-to-guides/12-param-schedulers.ipynb [hunk 1]
```diff
@@ -0,0 +1,835 @@
...
+        " "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import ast\n",
```
Comment (sdesrozis) [inline how-to-guides/12-param-schedulers.ipynb:31]: Useful ?

File: how-to-guides/12-param-schedulers.ipynb [hunk 2]
```diff
@@ -0,0 +1,835 @@
...
+        "    plt.show()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## LambdaLR\n",
+        "\n",
+        "Sets the learning rate of each parameter group to the initial lr times a given function. When last_epoch=-1, sets initial lr as lr.\n",
```
Comment (sdesrozis) [inline how-to-guides/12-param-schedulers.ipynb:115]: last_epoch ? Just copy / paste sounds not enough.

File: how-to-guides/12-param-schedulers.ipynb [hunk 3]
```diff
@@ -0,0 +1,835 @@
...
+        "\n",
+        "$$\n",
+        "l r_{\\text {epoch}} = l r_{\\text {initial}} * Lambda(epoch)\n",
```
Comment (sdesrozis) [inline how-to-guides/12-param-schedulers.ipynb:118]: lr is modified every step, not epoch. In the PyTorch doc, `optimizer.step()` is called at the epoch end, it’s more flexible in ignite. The formula should not be written using epoch as index name.

File: how-to-guides/12-param-schedulers.ipynb (hunk 1 above)
Comment (divyanshugit) [reply how-to-guides/12-param-schedulers.ipynb:31]: It is used in `MultiStepLR`  for parsing the string values of Milestones.

File: how-to-guides/12-param-schedulers.ipynb (hunk 3 above)
Comment (divyanshugit) [reply how-to-guides/12-param-schedulers.ipynb:118]: Oh, my bad. I'm sorry. I didn't think in this manner.

File: how-to-guides/12-param-schedulers.ipynb (hunk 2 above)
Comment (divyanshugit) [reply how-to-guides/12-param-schedulers.ipynb:115]: Should I try to rephrase it?

**Pull request closed without merging**